views/deductions/trillium-rent-amount.pug

```diff
@@ -1,7 +1,7 @@
 extends ../base
 
 block variables
-  -var title = __('Enter rent paid in 2018')
+  -var title = __('Enter your rent payments to apply for OTB')
```

Comment:
I'm not sure if they are actually "applying". I thought they would just be automatically eligible and we are asking for their rent payments to see how much they get back.

Can you check with @YedidaZalik?

Reply:
I used "apply" because of #4 in this: https://www.canada.ca/en/revenue-agency/services/child-family-benefits/provincial-territorial-programs/ontario-trillium-benefit-questions-answers.html where it says "To apply for the 2019 OEPTC and/or NOEC components on your 2018 return, fill out  . . ."  Is there a way to flag that for Nancy to double-check?